.gitignore

```diff
@@ -11,7 +11,6 @@
 /dist
 
 # misc
-.vscode
```

Comment:
Should still .gitignore other vscode files, eg:

```
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
!.vscode/*.code-snippets
```
(lifted from https://github.com/github/gitignore/blob/main/Global/VisualStudioCode.gitignore)